draft-ietf-quic-transport.md

```diff
@@ -513,12 +513,12 @@ implementations. An implementation can define a different state machine as long
 as its behavior is consistent with an implementation that implements these
 states.
 
-Note:
-
-: In some cases, a single event or action can cause a transition through
-  multiple states.  For instance, sending STREAM with a FIN bit set can cause
-  two state transitions for a sending stream: from the Ready state to the Send
-  state, and from the Send state to the Data Sent state.
+<aside><t>
```

Comment:
Try `<aside markdown="1">`, then you can use normal markdown inside the aside.

Reply:
Yeah, I've started doing that on other changes, but I need to review all the open PRs for it now.